docs/dasharo-tools-suite/releases.md

```diff
@@ -13,6 +13,34 @@ Process](../dev-proc/standard-release-process.md).
 
 [newsletter]: https://newsletter.3mdeb.com/subscription/ttzqCq9fy
 
+## v1.2.0 - 2023-04-20
+
+### Images
+
+[USB bootable DTS v1.2.0 image][USB_DTS_v1.2.0]{ .md-button }
+[sha256][USB_DTS_sha_v1.2.0]{ .md-button }
+[sha256.sig][USB_DTS_sig_v1.2.0]{ .md-button }
+
+[DTS v1.2.0 ISO image][ISO_DTS_v1.2.0]{ .md-button }
+[sha256 ISO][ISO_DTS_sha_v1.2.0]{ .md-button }
+[sha256.sig ISO][ISO_DTS_sig_v1.2.0]{ .md-button }
+
+[USB_DTS_v1.2.0]: https://3mdeb.com/open-source-firmware/DTS/v1.2.0/dts-base-image-v1.2.0.wic.gz
+[USB_DTS_sha_v1.2.0]: https://3mdeb.com/open-source-firmware/DTS/v1.2.0/dts-base-image-v1.2.0.wic.gz.sha256
+[USB_DTS_sig_v1.2.0]: https://3mdeb.com/open-source-firmware/DTS/v1.2.0/dts-base-image-v1.2.0.wic.gz.sha256.sig
+[ISO_DTS_v1.2.0]: https://3mdeb.com/open-source-firmware/DTS/v1.2.0/dts-base-image-v1.2.0.iso
+[ISO_DTS_sha_v1.2.0]: https://3mdeb.com/open-source-firmware/DTS/v1.2.0/dts-base-image-v1.2.0.iso.sha256
+[ISO_DTS_sig_v1.2.0]: https://3mdeb.com/open-source-firmware/DTS/v1.2.0/dts-base-image-v1.2.0.iso.sha256.sig
+
+  See how to verify hash and signature on [this
+  video](TBD-update!). It works the same way with ISO image.
```

Comment:
@TomaszAIR when do we plan to provide this video?

Reply:
For future I opt for consistent documentation and quality procedure there instead of video/asciinema. But for now let's follow the process we have.

Reply:
@macpijan after merging https://github.com/Dasharo/meta-dts/pull/33 I will sign release, provide video and then this PR also can be merged

Reply:
link provided, resolving